tests_e2e/playwright/README.md

```diff
@@ -0,0 +1,84 @@
+# Nebari integration testing with Playwright
+
+
+## How does it work?
+
+Playwright manages interactions with any website. We are using it to interact
+with a deployed Nebari instance and test the various integrations that are
+included.
+
+For our test suite, we utilize Playwright's synchronous API. The first task
+is to launch the web browser you'd like to test in. Options in our test suite
+are `chromium`, `webkit`, and `firefox`. Playwright uses browser contexts to
+achieve test isolation. The context can either be created by default or
+manually (for the purposes of generating multiple contexts per test in the case
+of admin vs user testing). Next the page on the browser is created. For all
+tests this starts as a blank page, then during the test, we navigate to a given
+url. This is all achieved in the `setup` method of the `Navigator` class.
+
+## Setup
+
+Install Nebari with the development requirements (which include Playwright)
+
+`pip install -e ".[dev]"`
+
+Then install playwright itself (rerquired).
+
+`playwright install`
+
+:::note
+If you see the warning `BEWARE: your OS is not officially supported by Playwright; downloading fallback build.`, it is not critical. Playwright will likely still
+work https://github.com/microsoft/playwright/issues/15124
```

Comment:
This doesn't render properly for me

![Screenshot from 2023-05-11 11-02-13](https://github.com/nebari-dev/nebari/assets/6849766/ddc5be02-66eb-4bd3-875c-11467932ab33)